docs/development.rst

```diff
@@ -195,19 +195,19 @@ reworked and resubmitted for the next release.
 Deprecation Policy
 ==================
 
-Any change to pip that removes or significantly alters user-visible behaviour
+Any change to pip that removes or significantly alters user-visible behavior
 that is described in the pip documentation will be deprecated for a minimum of
-one released version before the change occurs. Deprecation will take the form of
-a warning being issued by pip when the feature is used. Longer deprecation
-periods, or deprecation warnings for behaviour changes that would not normally
-be covered by this policy, are also possible depending on circumstances, but
-this is at the discretion of the pip developers.
+6 months before the change occurs. Deprecation will take the form of a warning
```

Comment:
@dstufft "one released version" is a 3 month period whereas this says 6 months. Is that intentional?

Reply:
(I'd noticed the 6 month earlier but I didn't realize the time frame got changed until now.)